### PR TITLE
Sync code to v7

### PR DESCRIPTION
**Pull request by bualoy-napat**
## Description

Sync code to v7

### CODE REVIEW

**Review by Sakchai-Refinitiv — DISMISSED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -35401,7 +35413,7 @@
       }
     },
     "pandora-book": {
-      "version": "file:tools/pandora-book-3.0.0-66.tgz",
+      "version": "file:tools\\pandora-book-3.0.0-66.tgz",
```
Comment (Sakchai-Refinitiv) [inline package-lock.json:35416]: Wh the directory separator changed

Comment (bualoy-napat) [reply package-lock.json:35416]: because of the window OS have different the path separator

Comment (bualoy-napat) [reply package-lock.json:35416]: However, I think this should not change. I will restore this point back the base version

Comment (bualoy-napat) [reply package-lock.json:35416]: @Sakchai-Refinitiv I already reverted this change back to the base version

**Review by Sakchai-Refinitiv — APPROVED** [review]

**Pull request merged**